plugins/index.md

```diff
@@ -11,14 +11,14 @@ Tools without plugins are going to be listed in the table under the `Tools`
 header. Tools with plugins will have their own header and table listing the
 plugins defined for that tool.
 
-<div class="panel panel-info">
-  <div class="panel-heading">Table of Contents</div>
+<div class="card">
+  <div class="card-header panel-heading bg-primary text-white">Table of Contents</div>
   <div class="panel-body">
-  <ul>
-  <li><a href="#tools">Tools</a></li>
+	<ul class="list-group">
+	<li class="list-group-item"><a href="#tools">Tools</a></li>
```

Comment:
Nit: the rest uses spaces and this uses tabs

Reply:
fixed it